.circleci/config.yml

```diff
@@ -0,0 +1,173 @@
+version: 2.1
+
+commands:
+  build_data_jar:
+    description: Package the standing data into a jar file
+    steps:
+      - run:
+          name: Package data with maven
+          command: mvn package --no-transfer-progress
+
+  checkout_bichard_repo:
+    description: Check out the bichard7-next repo
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - 43:12:ce:d2:03:6f:b4:8e:a5:c8:8e:f2:85:89:fc:7c
```

Comment:
thought: I wonder if it would be worth documenting these ssh keys in something like a google doc, we seem to have quite a few of these now and if they only exist in the code we might lose track of them if they get removed. We might already be doing this but I can't find anything relevant

Reply:
Hmmm, what do you suggest that documentation would look like?

In this case, this keypair is generated specifically for the purpose of being able to clone the bichard7-next repository from CircleCI. The private key is stored as a secret in CircleCI, and the public key is added to the GitHub repo settings as a deploy key. I think all of the other keys that I've seen like this are used in the same way.

Reply:
If it helps, the fingerprint referred to here in this config is taken directly from the CircleCI settings page: https://app.circleci.com/settings/project/github/ministryofjustice/bichard7-next-data/ssh